website/docs/r/logs_index.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "datadog"
+page_title: "Datadog: datadog_logs_index"
+sidebar_current: "docs-datadog-resource-logs-index"
+description: |-
+  Provides a Datadog logs index resource. This can be used to create and manage logs indexes.
+---
+
+# datadog_logs_index
+
+Provides a Datadog [Logs Index API](https://docs.datadoghq.com/api/?lang=python#logs-indexes) resource. This can be used to create and manage Datadog logs indexes.
+
+## Example Usage:
+Create a Datadog logs index resource.
```

Comment:
This line says the sample is to create a Datadog logs index resource, but the sample says "update"—is this intentional?

Reply:
@cswatt Thanks for the review. I wanted to say that by creating this resource we create a state in terraform, but this creation is going to UPDATE the index from backend.